packages/agent/conf/demos/knb/babylon/templates/babylon/BankAccount.hbs

```diff
@@ -7,8 +7,7 @@
         "BankAccount"
     ],
     "credentialSubject": {
-        "Omschrijving": "Bankrekening bij Babylon",
-        "Rekeningnummer": "NL31 BABY 0123456789",
```

Comment:
why was this removed?

Reply:
Omschrijving is reverted back to description
Rekeningnummer is auto-generated in the form now